lib/board.rb

```diff
@@ -0,0 +1,81 @@
+class Board
+  attr_reader :grid
+
+  def initialize(grid = [[1, 2, 3], [4, 5, 6], [7, 8, 9]])
+    @grid = grid
+  end
+
+  def get_value(position)
+    line, column = get_line_column(position)
+    @grid[line][column]
+  end
+
+  def get_line_column(position)
+    line = position[0]
+    column = position[1]
+    return line, column
+  end
+
+  def set_value(argument)
```

Comment:
Ruby language feature: You can destructure this argument directly eg `def set_value(position_label: nil, move: nil)`.